org.karapulse.Karapulse.json

```diff
@@ -75,6 +75,31 @@
                 "./install.sh --prefix=/app/sdk/rust --disable-ldconfig --verbose"
             ]
         },
+        {
+            "name": "python-2.7",
```

Comment:
Add Shared Modules
https://github.com/flathub/flathub/wiki/App-Requirements#shared-modules
https://github.com/flathub/shared-modules/tree/master/python2.7

Reply:
@bochecha suggested me to copy it so I can use ` "cleanup": ["*"] ` as I just need python for the build and not in the package.